src/extension/flutter/flutter_daemon.ts

```diff
@@ -50,6 +50,7 @@ export class FlutterDaemon extends StdIOService<UnknownNotification> implements
 
 		if (portFromLocalExtension) {
 			this.createNcProcess(portFromLocalExtension);
+			this.startPing();
```

Comment:
Should we start this only once we know the daemon is up and running? I'm wondering if otherwise it might fail before it's really set up, and then prompt for a restart?

Reply:
I think this should be okay because we only get a port from the local extension after the process has started. If the process has started but there's no response, then it would be appropriate to restart.